test/Confluent.SchemaRegistry.Serdes.UnitTests/Program.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Threading.Tasks;
+
+
+// namespace Confluent.SchemaRegistry.Serdes.UnitTests
```

Comment:
?

Reply:
removed. i was considering ditching using the test project type and switching to console instead because the former was annoying me, but then remembered it's value.